Feature/kubecon 2023 (#774)
ordina-jworks/ordina-jworks.github.io: _posts/2023-05-17-kubecon-2023.md

---
layout: post
authors: [yolan_vloeberghs, pieter_vincken, yannick_bontemps, lander_marien]
title: 'KubeCon + CloudNativeCon 2023'
image: /img/2023-05-17-kubecon-2023/banner-resized.jpg
tags: [cloud, conference, cloud-native, kubernetes]
category: Cloud
comments: true
---
# Introduction

{:refdef: style="text-align: center;"}
<img src="{{ '/img/2023-05-17-kubecon-2023/group-picture.jpg' | prepend: site.baseurl }}" alt="JWorks group picture" class="image" style="margin:0px auto; max-width:100%">
_JWorks at Kubecon 2023_
{: refdef}

Another year, another KubeCon | CloudNativeCon EU edition. 

> And of course, JWorks was also present at this year's edition in Amsterdam in the [RAI](https://www.rai.nl/){:target="_blank" rel="noopener noreferrer"}.

Reviewer: ```suggestion
And, of course, JWorks was also present at this year's edition in Amsterdam in the [RAI](https://www.rai.nl/){:target="_blank" rel="noopener noreferrer"}.
```

We wanted to hear more about all the new features, frameworks, tools, ideas and concepts that the Kubernetes and cloud world have to offer.
And we got what we wanted.
We attended many interesting talks, talked to very intriguing people at the event, and had a lot of fun while doing so.

> Next to the talks, there were also a large number of booths from companies from all over the world (AWS, Azure, Canonical, ...) to engage with other people and to talk and promote their newest products for the developer market.

Reviewer: ```suggestion
Next to the talks, there were also many booths from companies from all over the world (AWS, Azure, Canonical, ...) to engage with other people and to talk and promote their newest products for the developer market.
```

> With that being said, you can take a look at some of the talks we attended which we thought were very interesting.
> You can click on the title of the talk to go to the recorded version on YouTube, which is provided by CNCF.

Reviewer: ```suggestion
You can look at some of the talks we attended, which were very interesting.
You can click on the talk title to go to the recorded version on YouTube, which CNCF provides.
```
